examples/core-hello-world/src/main/resources/application.conf

```diff
@@ -1,10 +1,10 @@
 # This is still the default configuration but it's good to see what can be changed
 mesos-client {
 
-  master-url: "127.0.0.1:5050"
-  master-url: ${?MESOS_MASTER_URL}
+  master-url: [ "http://127.0.0.1:5050" ]
+  master-url: [ ${?MESOS_MASTER_URL} ]
```

Comment:
Does env var override work for arrays?

Reply:
To answer my own question - it does, but you can only pass **one** value.

Reply:
Oof, yeah. A quick search reveals that we probably need to resort to splitting for env-var overrides :'(

https://github.com/lightbend/config/issues/163